src/main/java/io/observatorium/opentelemetry/generate/account/TracerInitializer.java

```diff
@@ -1,44 +1,47 @@
 package io.observatorium.opentelemetry.generate.account;
 
+import java.util.concurrent.TimeUnit;
+
 import javax.enterprise.context.ApplicationScoped;
 import javax.enterprise.event.Observes;
 import javax.enterprise.inject.Produces;
 
-import io.opentelemetry.common.AttributeValue;
-import io.opentelemetry.common.Attributes;
-import io.opentelemetry.exporters.otlp.OtlpGrpcSpanExporter;
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.trace.Tracer;
+import io.opentelemetry.exporter.otlp.trace.OtlpGrpcSpanExporter;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
 import io.opentelemetry.sdk.resources.Resource;
 import io.opentelemetry.sdk.resources.ResourceAttributes;
-import io.opentelemetry.sdk.trace.TracerSdkProvider;
+import io.opentelemetry.sdk.trace.SdkTracerProvider;
 import io.opentelemetry.sdk.trace.export.BatchSpanProcessor;
-import io.opentelemetry.trace.Tracer;
 import io.quarkus.runtime.ShutdownEvent;
 import io.quarkus.runtime.StartupEvent;
 
 @ApplicationScoped
 public class TracerInitializer {
-    TracerSdkProvider sdkProvider;
+    OpenTelemetrySdk sdkProvider;
 
     @Produces
     @Account
     Tracer tracer;
 
     void onStart(@Observes StartupEvent ev) {
-        sdkProvider = TracerSdkProvider.builder().setResource(Resource.create(
-                Attributes.of(ResourceAttributes.SERVICE_NAME.key(), AttributeValue.stringAttributeValue("account"))))
-                .build();
+        OtlpGrpcSpanExporter spanExporter = OtlpGrpcSpanExporter.builder().setTimeout(2, TimeUnit.SECONDS).build();
+        BatchSpanProcessor spanProcessor = BatchSpanProcessor.builder(spanExporter)
+                .setScheduleDelay(100, TimeUnit.MILLISECONDS).build();
 
-        OtlpGrpcSpanExporter spanExporter = OtlpGrpcSpanExporter.getDefault();
-        BatchSpanProcessor spanProcessor = BatchSpanProcessor.newBuilder(spanExporter).setScheduleDelayMillis(100)
-                .build();
-        sdkProvider.addSpanProcessor(spanProcessor);
+        sdkProvider = OpenTelemetrySdk.builder()
```

Comment:
One little nit - shouldn't these 4 lines be aligned?  (from .setTracerProvider down to .buildAndRegisterGlobal())  

I haven't written much Java lately so I don't know what current conventions are.

Reply:
I haven't done so myself either, which is why I let the editor to format it for me :-)

Reply:
I suppose there's some reason the editor did it that way but I can't imagine what it was.  In any case, it's not a big deal to me.